src/internal/connector/exchange/api/events.go

```diff
@@ -244,33 +234,30 @@ func (c Events) GetAddedAndRemovedItemIDs(
 		return nil, nil, DeltaUpdate{}, err
 	}
 
-	var (
-		resetDelta bool
-		errs       *multierror.Error
-	)
+	var resetDelta bool
 
 	ctx = clues.AddAll(
 		ctx,
-		"category", selectors.ExchangeEvent,
 		"calendar_id", calendarID)
 
 	if len(oldDelta) > 0 {
-		builder := users.NewItemCalendarsItemEventsDeltaRequestBuilder(oldDelta, service.Adapter())
-		pgr := &eventPager{service, builder, nil}
+		var (
+			builder = users.NewItemCalendarsItemEventsDeltaRequestBuilder(oldDelta, service.Adapter())
+			pgr     = &eventPager{service, builder, nil}
+		)
 
 		added, removed, deltaURL, err := getItemsAddedAndRemovedFromContainer(ctx, pgr)
 		// note: happy path, not the error condition
 		if err == nil {
-			return added, removed, DeltaUpdate{deltaURL, false}, errs.ErrorOrNil()
+			return added, removed, DeltaUpdate{deltaURL, false}, err
```

Comment:
If err == nil, why would we return the error rather than nil?

Reply:
hah, I was on autopilot.  Good catch.